src/app/libs/sidebar/infrastructure/tokens.ts

```diff
@@ -1,4 +1,4 @@
-import { SidebarDataPort, ShowSettingsUiPort, ShowSettingsUseCase } from '../use-cases';
+import { SidebarDataPort, SidebarUiPort, SidebarUseCase } from '../use-cases';
 import { SidebarStoreService } from './sidebar-store.service';
 
 export const INFRASTRUCTURE_PROVIDERS = [
```

Comment:
I don't know how this is still working without providing `deps` array with `SettingsApi`. @hoebbelsB maybe you have an idea?

Reply:
Yes 😅 that's what I was curious about.